util/helm/repo/index_test.go

```diff
@@ -1,24 +0,0 @@
-package repo
```

Comment:
don't you want these tests, but move them?

Reply:
oh, nice catch. thank you!